### ISSUE TITLE
Wrong footnotes order (need update to docusaurus v3)

### ISSUE THREAD

**Issue by RezaRahemtola** [gnolang/docs.gno.land#49]
## Description

While reading the docs, I noticed that the [footnotes in this page](https://docs.gno.land/reference/go-gno-compatibility) where not correctly ordered / matching the footnote number:
- the [`time.Now` one](https://docs.gno.land/reference/go-gno-compatibility#fn-7) is the 7th but placed as 1st
- sort is the 6th but placed 2nd
- math is the 9th but placed 3rd
- ...

This was addressed here: https://github.com/facebook/docusaurus/issues/6625 (tldr: can be fixed either by using a [forked package dependency](https://github.com/facebook/docusaurus/issues/6625#issuecomment-1031162044) but not working for me locally, or upgrading to docusaurus v3)

This isn't a very important bug, but upgrading would be great imo to benefit from the latest MDX features (Docusaurus v3 upgrades MDX from v1 to v3).
Can help on this if needed ;)

**Comment by alexiscolin** [comment]
Yes, we already discussed upgrading Docusaurus to v3 and I already have a local branch with the V3 (to test all components). I think I'll push it soon.
Thank you for the report :)
